<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filter_search</class>
 <widget class="QDialog" name="filter_search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>393</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>308</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>3213213</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Filter search</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>311</width>
     <height>386</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>311</width>
     <height>386</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>311</width>
     <height>386</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
border-radius: 5px;
background-color: #2a2a2a;
color: white;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>200</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
color: white;</string>
    </property>
    <property name="text">
     <string>Описание</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>80</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
color: white;</string>
    </property>
    <property name="text">
     <string>Максимальная цена</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>120</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
color: white;</string>
    </property>
    <property name="text">
     <string>Минимальная цена</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>195</y>
      <width>176</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 2px 5px;
font-size: 14px;
background-color: #404040;
border-radius: 5px;</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>196</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
color: white;</string>
    </property>
    <property name="text">
     <string>Фильтры поиска</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>160</y>
      <width>106</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 15px;
color: white;</string>
    </property>
    <property name="text">
     <string>Технологии</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>115</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 2px 5px;
font-size: 14px;
background-color: #404040;
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>75</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 2px 5px;
font-size: 14px;
background-color: #404040;
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>345</y>
      <width>116</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 14px;
	background-color: #404040;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #949494;
}</string>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>155</y>
      <width>176</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 2px 5px;
font-size: 14px;
background-color: #404040;
border-radius: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Через ;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>315</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 14px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>345</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 14px;
	background-color: #404040;
	border-radius: 5px;
}
QPushButton::hover {
	background-color: #949494;
}</string>
    </property>
    <property name="text">
     <string>Сбросить фильтры</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border-bottom-left-radius: 0px;
	border-bottom-right-radius: 0px;
	background-color: #404040;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>5</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font-size: 20px;
	background-color: #f73643;
	border-radius: 8px;
}
QPushButton::hover {
	background-color: #d63a44;
	color: black;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>5</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	padding-bottom: 4px;
	font-size: 20px;
	background-color: #47fc65;
	border-radius: 8px;
}
QPushButton::hover {
	background-color: #49cc5f;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>5</y>
       <width>106</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 14px;
color: white;</string>
     </property>
     <property name="text">
      <string>Filter search</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
